composer.json

```diff
@@ -15,11 +15,11 @@
         "source": "https://github.com/stevegrunwell/time-constants"
     },
     "require": {
-        "php": "^7.0"
+        "php": "^7.0 | ^8.0"
     },
     "require-dev": {
-        "friendsofphp/php-cs-fixer": "^2.13",
-        "phpunit/phpunit": "^6.0"
+        "friendsofphp/php-cs-fixer": "^3.5",
+        "phpunit/phpunit": "^9.5"
```

Comment:
According to the [official PHPUnit doc](https://phpunit.de/announcements/phpunit-9.html), When upgrading PHPUnit to `9.x`, it will support `php-7.3` or newer version.

It will be failed on `php-7.0` version.

Reply:
This is true, but considering the library is a series of `define()` statements, running it across multiple versions of PHP (and thus PHPUnit) seems a little overkill :smile: